src/App.tsx

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { Suspense, lazy } from 'react';
 
 import { TbLayoutSidebarRightCollapseFilled } from 'react-icons/tb';
 
 import { Axe } from '@/components/Axe';
 import FilterSidebar from '@/components/FilterSidebar';
 import { useApp } from '@/hooks/useApp';
-import DataCatalogue from '@/pages/DataCatalogue';
-import MapViewer from '@/pages/MapViewer';
+
+const DataCatalogue = lazy(() => import('@/pages/DataCatalogue'));
+const MapViewer = lazy(() => import('@/pages/MapViewer'));
```

Comment:
yeah, it is great, but it might have come at a cost. I don't think the hook in to Django is liking it and I'm not knowledgeable enough on Django and the `ManifestStaticFilesStorage` stuff. The whole way we link Django and our stati catalogue assets is a bit of a mystery to me as I've never done it the way the web presence does.

Reply:
More for reference really, I'm going to revert the commit for this as it is breaking `dev`, I need to deploy this to dev and break it to figure it out, or maybe `dev2` would be more sensible as it will likely be more trial and error.